support-frontend/assets/helpers/routes.js

```diff
@@ -38,9 +38,12 @@ const routes: {
   guardianWeeklySubscriptionLandingGift: '/subscribe/weekly/gift',
   postcodeLookup: '/postcode-lookup',
   createSignInUrl: '/identity/signin-url',
-  createReminder: '/contribute/remind-me',
+  createReminder: 'https://contribution-reminders-code.support.guardianapis.com/remind-me',
```

Comment:
Will this still work @jlieb10?

Reply:
No that needs to be cleaned up - thank you

Reply:
using `createReminderEndpoint` (below) instead

Reply:
(which should probably be `getReminderEndpoint` actually)